frontend/src/bundles/home/components/home-top-section/components/create-resume-button/create-resume-button.tsx

```diff
@@ -0,0 +1,28 @@
+import plusCircleIcon from '~/assets/img/plusCircleIcon.svg';
+import resumeIcon from '~/assets/img/resumeIcon.svg';
+
+import styles from './styles.module.scss';
+
+const CreateResumeButton: React.FC = () => {
+    return (
+        <div className={styles.container}>
+            <div className={styles.create_resume_button}>
+                <div className={styles.create_resume_button__icon}>
+                    <img
```

Comment:
Don't use img for svg, you can use svg as a component

Reply:
doesn't it make html insanely ugly